go.mod

```diff
@@ -5,16 +5,15 @@ go 1.16
 require (
 	github.com/BurntSushi/toml v0.4.1
 	github.com/Masterminds/semver/v3 v3.1.1
+	github.com/anchore/syft v0.31.0
 	github.com/cheggaaa/pb/v3 v3.0.8
-	github.com/dsnet/compress v0.0.1
+	github.com/dsnet/compress v0.0.2-0.20210315054119-f66993602bf5
 	github.com/gabriel-vasile/mimetype v1.4.0
-	github.com/kr/text v0.2.0 // indirect
-	github.com/niemeyer/pretty v0.0.0-20200227124842-a10e7caefd8e // indirect
 	github.com/onsi/gomega v1.17.0
 	github.com/pelletier/go-toml v1.9.4
 	github.com/sclevine/spec v1.4.0
 	github.com/ulikunitz/xz v0.5.10
-	golang.org/x/net v0.0.0-20210805182204-aaa1db679c0d // indirect
-	golang.org/x/sys v0.0.0-20210809222454-d867a43fc93e // indirect
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 )
+
+// TODO: remove once a new release is cut with the CycloneDX format (probably v0.32.0)
+replace github.com/anchore/syft => github.com/anchore/syft v0.31.1-0.20211204010623-5374a1dc6ff6
```

Comment:
are we happy merging this into main?

Reply:
the alternative, i guess, is blocking on a syft release...

Reply:
I'm OK with it

Reply:
The other alternative is to add our own JSON conversion code, which just doesn't seem worth it at this point.